rfc/rfc004-package-log.md

```diff
@@ -12,8 +12,21 @@ http://peter.bourgon.org/go-kit/#package-log
 
 ## Scope
 
-To be defined.
+* Log SHALL be drop in compatible with the standard library logger.
+* Log SHALL use key / value pairs for structured data.
+* Log MAY allow multiple stored logging contexts each with a different set of k/v pairs.
+* Log SHALL be transport agnostic by accepting and io.Writer as output target.
+* Log SHALL provide a default stdout io.Writer.
+* Log MAY implement io.MultiWriters allowing broadcast of logs over multiple transports.
+* Log MAY provide configurable back pressure handling strategies in the case of blocked io.Writers
+* Log MAY provide some built in transports such as syslog and logstash.
+* Log SHALL be format agnostic by providing an interface for log formatting.
+* Log MAY provide some built in formatters such as RFC3164, JSON, etc.
+* Log SHALL provide a set of defined severity levels.
+* Log MUST NOT intrinsicly tie severity levels to program actions - e.g., a call to a specific log level should not call a panic.
```

Comment:
To me, "handling an error" and "logging about an error" are two distinct concerns.  I'm not a fan of things like log.Fatal and log.Panic, as I think terminating execution of a program is something that should be outside of the domain of the logger, which should just care about logging.

I'm open to discussion, just my opinion.